dbt/adapters/databricks/connections.py

```diff
@@ -58,6 +61,27 @@
 
 logger = AdapterLogger("Databricks")
 
+
+class DbtCoreHandler(logging.Handler):
+    def __init__(self, level: Union[str, int], *args: Tuple, **kwargs: Mapping):
+        super().__init__(level=level)
+        self.logger = AdapterLogger("databricks-sql-connector")
```

Comment:
Two things here:
1. Why

*args: Tuple, **kwargs: Mapping

if you're not using them for anything?
2. Why not pass the logger in?  I'm generally not a fan of instantiating objects outside of a [composition root](https://blog.ploeh.dk/2011/07/28/CompositionRoot/)

Reply:
Works for me!

Great point about args and kwargs. Removed them. I revised the composition to inject the AdapterLogger rather than instantiating it within `__init__`.